src/js/view/view.js

```diff
@@ -805,6 +814,8 @@ define([
             _showing = true;
             utils.removeClass(_playerElement, 'jw-flag-user-inactive');
 
+            _controlbar.checkCompactMode(_videoLayer.clientWidth);
+
```

Comment:
Now that we're sampling clientWidth in this method, we may want to debounce the 'mousemove' event handler that triggers this method. Or, find a better place to run this check and/or sample clientWidth.